example/index.js

```diff
@@ -1,4 +1,4 @@
-const { flow, act, check } = require("../worker/dist");
+const { flow, act, check, start } = require("../dist"); // or "automated-qa-sdk"
```

Comment:
The examples are out of scope for this milestone thankfully! But yeah, we'll soon change this to `require("automated-qa-sdk")` or similar.